package.json

```diff
@@ -114,12 +114,12 @@
     "microbundle": "^0.11.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^1.17.1",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
+    "react": "^16.9.0-alpha.0",
```

Comment:
I'm not quite sure whether we really want to test against `16.9.0` quite yet. It potentially will cause us to miss bugs in non-batching mode (I assume `16.9` is batching by default?) and async act sounds like it'd also batch some effects in testing, which might not be desirable 😅 

Any idea of what kind of impact this might have on us?

Reply:
I know the author of react-testing-library was pretty fond of upgrading since without it your console will be spammed with act warning, I can revert but we will have some console.errors in console (won't impact test outcome though)

Reply:
We do have a workaround somewhere in the repo where we filter out these errors 👍

Reply:
Aha, then I'll find that and use it. Generally the approach is good?

Reply:
It does work. It's still a little noisy but tbh I trust it more than a future React version where things possibly become more async while people are targeting stable 😅

Reply:
Allright, I agree though the less risk the better didn't think about it that way yet